api-rest-service/src/main/java/edu/mit/ll/em/api/rs/impl/UserServiceImpl.java

```diff
@@ -110,17 +86,22 @@
 	private static String SUCCESS = "success";

```

Comment:
I do not think the constants on lines 76-87 are used any longer. Can they be removed?